example_code/shmem_quiet_example.c

```diff
@@ -1,27 +1,28 @@
-#include <stdio.h>
 #include <shmem.h>
+#include <stdio.h>
 
-int main(void)
-{
-   static long dest[3];
-   static long source[3] = { 1, 2, 3 };
-   static int targ;
-   static int src = 90;
-   long x[3] = { 0 };
-   int y = 0;
-   shmem_init();
-   int me = shmem_my_pe();
-   if (me == 0) {
-      shmem_put(dest, source, 3, 1); /* put1 */
-      shmem_put(&targ, &src, 1, 2);  /* put2 */
-      shmem_quiet();
-      shmem_get(x, dest, 3, 1);   /* gets updated value from dest on PE 1 to local array x */
-      shmem_get(&y, &targ, 1, 2); /* gets updated value from targ on PE 2 to local variable y */
-      printf("x: { %ld, %ld, %ld }\n", x[0], x[1], x[2]); /* x: { 1, 2, 3 } */
-      printf("y: %d\n", y); /* y: 90 */
-      shmem_put(&targ, &src, 1, 1); /* put3 */
-      shmem_put(&targ, &src, 1, 2); /* put4 */
-   }
-   shmem_finalize();
-   return 0;
+int main(void) {
+  static long dest[3];
+  static long source[3] = {1, 2, 3};
+  static int targ;
+  static int src = 90;
+  long x[3] = {0};
+  int y = 0;
+  shmem_init();
+  int me = shmem_my_pe();
+  if (me == 0) {
+    shmem_put(dest, source, 3, 1); /* put1 */
+    shmem_put(&targ, &src, 1, 2);  /* put2 */
+    shmem_quiet();
+    shmem_get(x, dest, 3,
+              1); /* gets updated value from dest on PE 1 to local array x */
+    shmem_get(&y, &targ, 1,
+              2); /* gets updated value from targ on PE 2 to local variable y */
+    printf("x: { %ld, %ld, %ld }\n", x[0], x[1], x[2]); /* x: { 1, 2, 3 } */
+    printf("y: %d\n", y);                               /* y: 90 */
+    shmem_put(&targ, &src, 1, 1);                       /* put3 */
+    shmem_put(&targ, &src, 1, 2);                       /* put4 */
+  }
```

Comment:
Some codes may require moving the comments to their own line (before the `shmem_get`) to auto-break more appealingly through clang-format.